nixos/modules/services/web-servers/hydron.nix

```diff
@@ -0,0 +1,111 @@
+{ config, lib, pkgs, ... }:
+
+let cfg = config.services.hydron;
+in with lib; {
+  options.services.hydron = {
+    enable = mkEnableOption "hydron";
+
+    dataDir = mkOption {
+      type = types.path;
+      default = "/var/lib/hydron";
+      example = "/home/okina/hydron";
+      description = "Location where hydron runs and stores data.";
+    };
+
+    interval = mkOption {
+      type = types.string;
+      default = "hourly";
+      example = "06:00";
+      description = ''
+        How often we run hydron import and fetch_tags. Runs by default every hour.
+
+        The format is described in
+        <citerefentry><refentrytitle>systemd.time</refentrytitle>
+        <manvolnum>7</manvolnum></citerefentry>.
+      '';
+    };
+
+    listenAddress = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      example = "127.0.0.1:8010";
+      description = "Listen on a specific IP address and port.";
+    };
+
+    importPaths = mkOption {
+      type = types.nonEmptyListOf types.path;
+      default = [ (cfg.dataDir + "/images") ];
```

Comment:
I think it would be better to use `types.listOf types.path`, `default = []`, and always try to import `dataDir + "/images"`. if the user doesn't want to use it, they can just not put pictures in there, so this option is reserved for user specified directories.